EasyPost.Tests/ServicesTests/ScanFormServiceTest.cs

```diff
@@ -44,7 +44,7 @@ public async Task TestAll()
 
             List<ScanForm> scanForms = scanFormCollection.ScanForms;
 
-            Assert.True(scanFormCollection.HasMore);
+            // Assert.True(scanFormCollection.HasMore);  // Not guaranteed to have more
```

Comment:
mmm it should always have more?

Reply:
The cassette I recorded didn't return more than 20 scan forms/didn't have a second page, so `HasMore` was false.

Reply:
Well, this assertion is wrong, we really want to be asserting the existence of the key. If C# doesn't even populate the key if it's false, then let's just remove this instead of commenting it out.

Reply:
It will populate the key one way or another (either with the value from JSON or `false` by default if the JSON key does not exist). The `HasMore` boolean will always be **something**

Reply:
Ok, then yeah let's change the bad assertion to test that the key is there. This is important because in some libs we were previously not returning this key so we want to protect against the regression to ensure people can paginate. The content of said key isn't important.

Reply:
Yeah, type-strict languages are always going to have the property, versus dynamic languages were we don't know if the property exists.